### PR TITLE
Custom tags for saved pocket links

### PR DESCRIPTION
**Pull request by andreasgrill**
Hi!
I've added an additional keyword trigger to manually add custom tags for saved links.
Here is an example:
`pocket add myTag1 myTag2`

I hope you like it.

Cheers,
Andreas

### CODE REVIEW

**Comment by fniephaus** [comment]
Hi @andreasgrill,
Thanks for the pull request. This is pretty cool!
I've already pushed a new release ([v4.3](https://github.com/fniephaus/alfred-pocket/releases/tag/v4.3))!

Cheers,
Fabio

File: src/pocket_save.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def main(wf):
     current_app = os.popen(
         """ osascript -e 'application (path to frontmost application as text)' """).readline().rstrip()
     if current_app in ['Google Chrome', 'Safari']:
-        if not add_item(get_browser_item(current_app)):
+        if not add_item(get_browser_item(current_app, wf.args)):
```
Comment (fniephaus) [inline src/pocket_save.py]: This probably should have been:
```python
if not add_item(get_browser_item(current_app), wf.args):
```
:smiley:

**Comment by andreasgrill** [comment]
Oops :)
Sorry, that must have been a copy/paste error. I programmed it directly in my workflow directory and later copied the changes to my local repo. I also didn't consider the calling add_item with a link from the clipboard. It's good that you added that too.

File: src/pocket_save.py (hunk 1 above)
Comment (andreasgrill) [inline src/pocket_save.py]: Oops :)
Sorry, that must have been a copy/paste error. I programmed it directly in my workflow directory and later copied the changes to my local repo. I also didn't consider the calling add_item with a link from the clipboard. It's good that you added that too.

Comment (fniephaus) [inline src/pocket_save.py]: No worries. I really like the idea!

**Pull request merged**